frontend/src/v4/routes/teamspaces/components/uploadModelFileDialog/uploadModelFileDialog.component.tsx

```diff
@@ -93,6 +108,18 @@ export class UploadModelFileDialog extends React.PureComponent<IProps, IState> {
 		const { modelId, teamspaceName, fetchModelSettings, fetchRevisions } = this.props;
 		fetchRevisions(teamspaceName, modelId, true);
 		fetchModelSettings(teamspaceName, modelId);
+		const browserTimezone = Intl.DateTimeFormat().resolvedOptions().timeZone;
+
+		const timezones = allTimezones.map((tz) => {
```

Comment:
with L120 and this we're looping the timezones twice. 

We could just deduce the default timezone whilst we're restructuring the data